reference/entry.json

```diff
@@ -2371,7 +2371,7 @@
             "description": "No Content. The organizer contact point has been updated successfully."
           },
           "400": {
-            "description": "Bad Request. Possible error types:\n\n* https://api.publiq.be/probs/url/not-found\n* https://api.publiq.be/probs/body/missing\n* https://api.publiq.be/probs/body/invalid-syntax\n* https://api.publiq.be/probs/body/invalid-data",
+            "description": "Bad Request. Possible error types:\n\n* https://api.publiq.be/probs/body/missing\n* https://api.publiq.be/probs/body/invalid-syntax\n* https://api.publiq.be/probs/body/invalid-data",
```

Comment:
I think you might have to merge the contact point branch into this one again. Or there's something else going on here that needs to be fixed.

Reply:
I removed `https://api.publiq.be/probs/url/not-found\` because that is part of the 404 error.

Reply:
Oh right, good catch!